internal/statestore/redis_indices.go

```diff
@@ -0,0 +1,99 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package statestore
+
+import (
+	"net/url"
+
+	structpb "github.com/golang/protobuf/ptypes/struct"
+	"github.com/sirupsen/logrus"
+	"open-match.dev/open-match/internal/config"
+	"open-match.dev/open-match/pkg/pb"
+)
+
+type indexedFields struct {
```

Comment:
Would it be more clear if we define an alias ‘type indexedFields map[string]float64’ here instead of defining a new struct?

Reply:
In a future PR, I also want to return a list of indexes that were used (so delete indexes will work properly.)  That's why I'm not just returning the map.